energy-integration/mex-governance/src/views.rs

```diff
@@ -0,0 +1,40 @@
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+use crate::{
+    config::{self, FarmVoteView},
+    events,
+};
+
+#[multiversx_sc::module]
+pub trait ViewsModule:
+    config::ConfigModule
+    + events::EventsModule
+    + energy_query::EnergyQueryModule
+    + week_timekeeping::WeekTimekeepingModule
+{
+    #[view(getAllWeekEmissions)]
+    fn get_all_week_emissions(&self, week: usize) -> MultiValueEncoded<FarmVoteView<Self::Api>> {
```

Comment:
Don't use a struct just for the view. Unnecessary encoding and a bunch of extra code for not reason. Do this instead:
```
pub type FarmVoteView<M> = MultiValue2<ManagedAddress<M>, BigUint<M>>;
```

And use this type instead of the struct in your view return.

Reply:
Added a new type.